src/main/java/be/syntra/devshop/DevshopBack/services/SearchServiceImpl.java

```diff
@@ -1,19 +1,27 @@
 package be.syntra.devshop.DevshopBack.services;
 
 import be.syntra.devshop.DevshopBack.entities.Product;
+import be.syntra.devshop.DevshopBack.models.ProductPageAndMinMaxPrice;
 import be.syntra.devshop.DevshopBack.models.SearchModel;
 import lombok.extern.slf4j.Slf4j;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.domain.Sort;
 import org.springframework.stereotype.Service;
 import org.springframework.util.StringUtils;
 
-import java.util.Comparator;
-import java.util.List;
-import java.util.stream.Collectors;
+import java.math.BigDecimal;
 
 @Slf4j
 @Service
 public class SearchServiceImpl implements SearchService {
+    @Value("${pagination.default.pageSize}")
+    private int paginationDefaultPageSize;
```

Comment:
defaultPageSize and defaultPageNumber perhaps?

Reply:
yes, replaced with suggestions, i tend to go overboard on descriptiveness sometimes